### PR TITLE
Fork basic circleci configuration here

### PR DESCRIPTION
**Pull request by agjohnson**
Noticed this was missing. Just a basic config forked from another library, nothing special.

### CODE REVIEW

**Comment by agjohnson** [comment]
It needs a python image layer as well, due to node-gyp

**Comment by ericholscher** [comment]
I'm 👍 on more automation. I think @davidfischer is probably most qualified to review this tho. Happy to dive in if there's something useful I can do.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+commands:
+  run-checks:
+    description: "Run basic checks"
+    steps:
+      - checkout
+      - run: npm ci
+      - run: npm run lint
+  run-build:
+    description: "Ensure compiled assets are up to date and HTML is valid"
```
Comment (davidfischer) [inline .circleci/config.yml:11]:
```suggestion
    description: "Ensure compiled assets are up to date"
```

This doesn't look like it verifies valid HTML.

**Review by davidfischer — APPROVED** [review]
Thanks for doing this. Looks great!

**Pull request merged**